packages/amplify_datastore/example/lib/codegen/Post.dart

```diff
@@ -105,7 +105,7 @@ class Post extends Model {
     buffer.write("id=" + id + ", ");
     buffer.write("title=" + title + ", ");
     buffer.write("rating=" + rating.toString() + ", ");
-    buffer.write("created=" + formatDateToISO8601(created));
+    buffer.write("created=" + created.toString());
```

Comment:
Are we making this change in the modelgen as well? I preferred ISO8601 format even in toString() since that's what users have to pass in the query predicates `Post.CREATED.eq("2020-02-02T20:20:20-08:00")`

Reply:
The modelgen team is using the toString method instead of formatDateToISO8601 

We can switch to the different format you suggested instead then and communicate that change to the modelgen team. 

If we do so could we use this method instead though?  I think it's a bit easier to read: 

`created.toIso8601String()`

Reply:
If it produces the same output, yes!